### ISSUE TITLE
Proxies Not Bypassing The Bandwidth Limit 

### ISSUE THREAD

**Issue by DJboutit** [tonikelope/megabasterd#401]
I have this problem it looks like the proxies are not bypassing Mega's bandwidth limit when Megabastered hits the program stops and starts downloading again when the bandwith timer is reset.  Time to switch the program to better proxies.

**Comment by leofrandsouza1** [comment]
Same here not bypassing Mega s bandwidth limit

**Comment by Rebaxa007** [comment]
Yes this happening. After downloading of 5 GB download speed reduced to just about 60 - 70 kbps. This is a major problem as the main purpose of this app is to bypass mega limits .

**Comment by DJboutit** [comment]
What you have to do is find your own proxies the ones supplied with the program are garbage

**Comment by Rebaxa007** [comment]
> What you have to do is find your own proxies the ones supplied with the program are garbage

I am using proxy from proxyscarpe but not working

**Comment by leofrandsouza1** [comment]
Wish we could have the ETA of file joining on the Application like a count
down or some thing.

I am kind a new to this github not sure where to raise the feature request

**Comment by QziP22** [comment]
Having the same problem, anybody find a solution?

**Comment by gohamstergo** [comment]
proxies found elsewhere are working fine for me. probably a problem with your chosen proxies, not the app.

**Comment by ashes-xda** [comment]
check this tutorial https://www.youtube.com/watch?v=bBd6qG0mnZY working fine for me

**Comment by renatodru** [comment]
> check this tutorial https://www.youtube.com/watch?v=bBd6qG0mnZY working fine for me

This video is private.
